### PR TITLE
Fix php config

### PR DESCRIPTION
**Pull request by mavimo**

### CODE REVIEW

File: ansible/roles/vagrant_local/tasks/main.yml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 ---
-- name: Set the hostname in /etc/hostname
-  shell: echo {{ vagrant_local.vm.hostname }} > /etc/hostname
```
Comment (omissis) [inline ansible/roles/vagrant_local/tasks/main.yml:3]: why did you remove this task?

File: ansible/vars/production.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ server:
     locale: en_US.UTF-8
 apache:
     install: '1'
-    docroot: /var/www
-    servername: milan2016.drupaldays.stage
+    docroot: /var/www/milan2016.drupaldays.org/web
```
Comment (omissis) [inline ansible/vars/production.yml:9]: +1 for the /milan2016.drupaldays.org part, but why /web?

File: ansible/vars/staging.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ server:
     locale: en_US.UTF-8
 apache:
     install: '1'
-    docroot: /var/www
+    docroot: /var/www/milan2016.drupaldays.stage/web
```
Comment (omissis) [inline ansible/vars/staging.yml:9]: same as above

File: ansible/vars/vagrant.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ vagrant_local:
     vm: { base_box: trusty64, hostname: default, ip: 192.168.88.88, memory: '1024', sharedfolder: ../website, useVagrantCloud: '1', syncType: nfs }
 apache:
     install: '1'
-    docroot: /var/www/milan2016.drupaldays.loc
+    docroot: /var/www/milan2016.drupaldays.loc/web
```
Comment (omissis) [inline ansible/vars/vagrant.yml:12]: same as above

File: ansible/roles/vagrant_local/tasks/main.yml (hunk 1 above)
Comment (mavimo) [inline ansible/roles/vagrant_local/tasks/main.yml:3]: Use the appropriate ansible module ([`hostname`](http://docs.ansible.com/ansible/hostname_module.html)) to set the hostname should make this step not required.

File: ansible/vars/production.yml (hunk 2 above)
Comment (mavimo) [inline ansible/vars/production.yml:9]: See https://github.com/drupaldevdays/website/commit/7b47cc8bf6ebd5c09cc8fd38461c1005ca530115

File: ansible/vars/staging.yml (hunk 3 above)
Comment (mavimo) [inline ansible/vars/staging.yml:9]: See https://github.com/drupaldevdays/website/commit/7b47cc8bf6ebd5c09cc8fd38461c1005ca530115

File: ansible/vars/vagrant.yml (hunk 4 above)
Comment (mavimo) [inline ansible/vars/vagrant.yml:12]: See https://github.com/drupaldevdays/website/commit/7b47cc8bf6ebd5c09cc8fd38461c1005ca530115

**Comment by omissis** [comment]
Deprecated by https://github.com/drupaldevdays/provisioning/pull/4
